magento/pwa-studio
# Shows Error on invalid Product Quantity

## Description

This PR pops up an error toast when a user enters a number into the product quantity stepper on the cart page that results in a graphQL error, usually occurring when the number entered exceeds the available stock of the product.

## Related Issue





Closes PWA-377.

## Acceptance 



### Verification Stakeholders


@schensley @sirugh 

### Specification



### Verification Steps


1. Ensure an item is in your cart
1. Go to the `/cart` page
1. Manually enter a very large number into the quantity stepper input text field
1. Remove focus from the quantity stepper input text field
1. See that an error toast appears with a sensible error message

## Screenshots / Screen Captures (if appropriate)

![Screen Shot 2020-02-26 at 12 02 50 PM](https://user-images.githubusercontent.com/13182778/75373446-f27bf900-588f-11ea-9156-aa35a74776ee.png)

![Screen Shot 2020-02-26 at 12 02 55 PM](https://user-images.githubusercontent.com/13182778/75373456-f90a7080-588f-11ea-9791-b80ee74fc547.png)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.

## Notes / Risks

The error message comes straight from the GraphQL backend. "Qty" isn't a great thing to show an end user, but we don't want to be in the business of massaging these error strings.
